library/src/main/java/com/chuckerteam/chucker/internal/support/DialogUtils.kt

```diff
@@ -0,0 +1,25 @@
+package com.chuckerteam.chucker.internal.support
+
+import android.content.Context
+import com.chuckerteam.chucker.internal.data.model.DialogData
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+
+internal object DialogUtils {
+    fun showDialog(
```

Comment:
We might change this to be an extension function for Context.

Reply:
updated as an extension  `ContextExt`